README.CentOS7.md

```diff
@@ -42,6 +42,24 @@ MySQL is not recommended/supported on CentOS, see [paragraph 2 here](https://www
 
 *Note:* LORIS developers (those NOT working with a .zip release codebase) should skip steps relating to hosting their database locally. Contact your system administrator for database credentials.
 
+### Configure MariaDB repository
+Configure MariaDB repository by manually creating the file `/etc/yum.repos.d/MariaDB.repo` with the following details:
+
+```
+[mariadb]
+name = MariaDB
+baseurl = http://yum.mariadb.org/10.3/centos7-amd64
+gpgkey=https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
+gpgcheck=1
+```
+
+Install PGP key with:
+```
+sudo rpm --import https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
+```
+
+Then install the database client and server packages.
```

Comment:
```suggestion
### Database client and server packages.
```

replace lines 67-74 with
`### Starting MariaDB server`

That should complete the change.

Reply:
@PapillonMcGill added your comment